zoo-css-challenge/style.css

```
@@ -64,6 +79,19 @@ button {
   background-color: #372772;
   border: 2px solid black;
   color: #fff;
+
+  /* 6) Learning: Increased font size from 1rem (16px) to 1.25rem (20px) */
+  font-size: 1.25rem;
+}
+
+/* 6) Learning: Used the focus and hover pseudo-classes to create 
+      a new rule for the button selector that changes the background and border colours 
+      when focused or hovered over, also slows the transition effect making it smoother :) */
+button:focus,
+button:hover {
```

```suggestion
.education-container button:focus,
.education-container button:hover {
```
The change of colour on focus/hover is only for the learning section, so we should scope it to only there